cmd/fpga_crihook/main.go

```diff
@@ -164,10 +189,29 @@ func (he *hookEnv) programBitStream(fpgaRegion string, fpgaAfu string, fpgaBitSt
 	return nil
 }
 
+func (he *hookEnv) getProgrammedAfu(deviceNum string) (string, error) {
+	aFuIDPath := fmt.Sprintf(he.aFUIDTemplate, deviceNum)
+	data, err := ioutil.ReadFile(aFuIDPath)
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(data)), nil
+}
+
 func (he *hookEnv) process(reader io.Reader) error {
-	fpgaRegion, fpgaAfu, err := he.getFPGAParams(reader)
+	fpgaRegion, fpgaAfu, deviceNum, err := he.getFPGAParams(reader)
```

Comment:
The `fpga` prefix in `fpgaAfu` is redundant. It's already clear that we deal with FPGA here.